CMake/fletch-tarballs.cmake

```diff
@@ -469,26 +469,54 @@ if(NOT WIN32)
 endif()
 
 # Protobuf
-if(NOT WIN32)
-  if (fletch_ENABLE_Protobuf OR fletch_ENABLE_ALL_PACKAGES OR AUTO_ENABLE_CAFFE_DEPENDENCY)
-    set(Protobuf_SELECT_VERSION "2.5.0" CACHE STRING "Select the  version of ProtoBuf to build.")
-    set_property(CACHE Protobuf_SELECT_VERSION PROPERTY STRINGS "2.5.0" "3.4.1")
+
+if (fletch_ENABLE_Protobuf OR fletch_ENABLE_ALL_PACKAGES OR AUTO_ENABLE_CAFFE_DEPENDENCY)
+  set(Protobuf_SELECT_VERSION "3.6.0.1" CACHE STRING "Select the  version of ProtoBuf to build.")
+  if(MSVC)
+    set_property(CACHE Protobuf_SELECT_VERSION PROPERTY STRINGS "2.7.0" "3.4.1" "3.6.0.1")
+  else()
+  set_property(CACHE Protobuf_SELECT_VERSION PROPERTY STRINGS "2.5.0" "2.6.1" "2.7.0" "3.4.1" "3.6.0.1")
   endif()
+endif()
+
+set(Protobuf_version ${Protobuf_SELECT_VERSION})
+set(Protobuf_url "https://github.com/google/protobuf/archive/v${Protobuf_SELECT_VERSION}.zip" )
 
-  set(Protobuf_version ${Protobuf_SELECT_VERSION})
+if (Protobuf_version VERSION_EQUAL 2.5.0)
+  if(MSVC)
+    message( FATAL_ERROR "Only Protobuf 2.7.0 and later is not supported in MSVC" )
```

Comment:
"Only ... is *not* supported"? This sentence reads very strangely, is it correct? (It looks like the "not" may be erroneous?)

(Same thing a couple lines down.)